src/const.ts

```diff
@@ -102,7 +102,8 @@ export const GP_ORDER_TX_HASHES = {
   4: 'GP_ORDER_TX_HASHES_4',
 }
 
-export const LIQUIDITY_TOKEN_LIST = new Set(['USDT', 'TUSD', 'USDC', 'PAX', 'GUSD', 'DAI'])
+const LIQUIDITY_TOKEN_LIST_VALUES = process.env.LIQUIDITY_TOKEN_LIST || 'USDT,TUSD,USDC,PAX,GUSD,DAI,sUSD'
+export const LIQUIDITY_TOKEN_LIST = new Set(LIQUIDITY_TOKEN_LIST_VALUES.split(',').map(symbol => symbol.trim()))
```

Comment:
`trim` just in case?

Reply:
that makes me think, can there be token symbols with a space?
yes there can
but probably not starting or ending with (trimmable) space, that would be ridiculous

Reply:
yes, because anyone can add a comma separated list of tokens. So in case there's a space somewhere, it won't break it

Reply:
what if there's a token with comma in symbol, `T,KEN`, the horror :scream:

Reply:
What if the token is a whitespace??

Reply:
>What if the token is a whitespace?? 
Doesn't deserve to be listed.